routes/index.js

```diff
@@ -58,9 +58,21 @@ router.post('/register', function(req, res, next) {
       return res.redirect('/register');
     }
 
+    if (!req.body.first_name) {
+      req.flash('error', 'Please enter your first name.')
+      return res.redirect('/register');
+    }
+
+    if (!req.body.last_name) {
+      req.flash('error', 'Please enter your last name.')
+      return res.redirect('/register');
+    }
+
```

Comment:
Sorry, commented above on this.

_@mistakevin That error message doesn't work. Or, at least, it doesn't in development. Because user.save() returns undefined, and that causes my localhost app to crash._